src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/shared/BaseDialogWrapper.kt

```diff
@@ -241,6 +241,7 @@ abstract class BaseDialogWrapper(
             iconComboBox(availableStartupProjects, { commonOptions.startupProject }, ::startupProjectSetter)
                 .validationOnInput(validator.startupProjectValidation())
                 .validationOnApply(validator.startupProjectValidation())
+                .comment("Your project is not listed? <a href='https://plugins.jetbrains.com/plugin/18147-entity-framework-core-ui/f-a-q#why-i-cant-see-my-project-in-a-startup-projects-field'>Help</a>")
```

Comment:
A little suggestion:

```suggestion
                .comment(
                    "Your project is not listed? " +
                    "<a href='https://plugins.jetbrains.com/plugin/18147-entity-framework-core-ui/f-a-q#why-i-cant-see-my-project-in-a-startup-projects-field'>" +
                        "Help" +
                    "</a>")
```